src/components/task/TaskPanel.tsx

```diff
@@ -84,6 +89,33 @@ export class TaskPanel extends React.Component<Props, State> {
 
   public render() {
 
+    const TaskDetails = () => (
+      <div>
+        <Typography>
+          <ClockIcon className="task-detail-icon" />
+          {this.stringifyNanoseconds(this.props.task.time_taken_ms * 1e6, true, true)}
```

Comment:
Instead of doing this, can you make the function take milliseconds?

Reply:
Fixed